src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/Period.scala

```diff
@@ -35,7 +35,8 @@ sealed class PeriodType(val sizeInMilliseconds: Long, format: String, truncateFi
     if (timeZone.isDefined) {
       sdf.setTimeZone(timeZone.get)
     }
-    sdf.format(date)
+    val formattedDate = sdf.format(date)
+    this.periodTypeName + "-" + formattedDate
```

Comment:
Can s-strings make this more readable? `s"${this.periodTypeName}-$formattedDate"`